packages/esm-bed-management-app/src/bed-administration/bed-administration-form.component.tsx

```diff
@@ -17,21 +17,29 @@ import {
   Select,
   SelectItem,
   Stack,
-  TextArea,
   TextInput,
 } from '@carbon/react';
 import { useTranslation } from 'react-i18next';
-import { getCoreTranslation, type Location } from '@openmrs/esm-framework';
+import { getCoreTranslation, translateFrom, type Location } from '@openmrs/esm-framework';
+import { moduleName } from '../index';
 import { type BedAdministrationData } from './bed-administration-types';
-import { type BedType, type BedFormData } from '../types';
+import type { BedType, BedWithLocation } from '../types';
 import styles from '../modals.scss';
 
+{
+  /**
+   * Adds translation for occupancy status options
+   * t('occupancyStatusAvailable', 'Available')
+   * t('occupancyStatusOccupied', 'Occupied')
+   */
+}
+
```

Comment:
These translation strings for some reason don't get picked up. Any help is appreciated :)

Reply:
Not sure - maybe because at this point in the code you have not yet declared `t` ?  What happens if you move this block somewhere in the code after you have the `const { t } = useTranslation();` line?

Reply:
Remove the parentheses around the comment sting.

Reply:
Thank you!